app/src/main/java/com/manna/presentation/sign_up/ProfileConfirmFragment.kt

```diff
@@ -0,0 +1,48 @@
+package com.manna.presentation.sign_up
+
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import androidx.activity.OnBackPressedCallback
+import com.manna.R
+import com.manna.common.BaseFragment
+import com.manna.databinding.FragmentProfileConfirmBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class ProfileConfirmFragment :
+    BaseFragment<FragmentProfileConfirmBinding>(R.layout.fragment_profile_confirm) {
+
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        val callback: OnBackPressedCallback = object : OnBackPressedCallback(
+            true
+        ) {
+            override fun handleOnBackPressed() {
+                val fragment =
+                    parentFragmentManager.fragments.findLast { it !is ProfileConfirmFragment }
```

Comment:
```suggestion
                    parentFragmentManager.fragments.findLast { 
                        it !is ProfileConfirmFragment && it is BaseFragment<*>
                    }
```
이거 이렇게 해야겠더라구요 시스템에서 사용하는 다른 Fragment가 반환될 수 있더라는